Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -192,33 +197,41 @@ public void ClimbCheck()
 		{
 			if (climbCheckTimer > climbCheckTime)
 			{
-				RaycastHit hit;
-				Physics.SphereCast( new Ray( transform.position, model.forward ), climbCheckRadius, out hit, climbCheckDistance, climbLayer );
-				if ( hit.transform )
-				{
-					StartClimbing( hit.collider );
-				}
-				else
+				if (_actorStats.CanUseStamina())
```

Comment:
Yes because otherwise your stamina will run out and you could have 0.X seconds of leeway time before you fall (which probably feels good, but we should be intentional with that sort of thing).